py/desimodel/test/test_io.py

```diff
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 """Test desimodel.io.
 """
-import os
+import os, sys
 import uuid
```

Comment:
I recommend using [tempfile](https://docs.python.org/3/library/tempfile.html) instead of rolling your own with `uuid`.  For example, see this [desisurvey unit test](https://github.com/desihub/desisurvey/blob/master/py/desisurvey/test/test_config.py).